styles.css

```
@@ -0,0 +1,76 @@
+body {
+    background-color: pink;
+}
+
+@media(max-width: 992px) {
+    body {
+        background-color: purple;
+    }
+}
+
+@media(max-width: 768px) {
+    body {
+        background-color: lightseagreen;
+    }
+}
+
+@media(max-width: 576px) {
+    body {
+        background-color: lightcoral;
+    }
+}
+
+img {
+    border: 5px solid rgb(0, 0, 0);
+    border-radius: 10px;
+  }
+
+  
+.container {
+    width: 1080px; margin-left: auto; margin-right: auto;
+}
+
+.row {
+    display:flex;
+}
+
+.column {
+    width: 50%
+}
+
+#secondbox {
+    border: 2px solid rgb(66, 64, 66);
+    border-radius: 5px;
+  }
+
+  #firstbox {
+    border: 2px solid rgb(66, 64, 66);
+    border-radius: 5px;
+  }
+
+h1 
+  {
+    font-weight: bolder;
+    text-align: center;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif
+}
+
+h2
+ {
+    font-weight: bold;
+    text-align: center;
+    font-family:Cambria, Cochin, Georgia, Times, 'Times New Roman', serif
+}
+
+#Projects {
```

This is a nitpick, but I would expect ids and classes to have consistent naming. 

```suggestion
#projects {
```